src/test/java/com/rafaskoberg/gdx/typinglabel/TypingLabelTest.java

```diff
@@ -180,6 +187,60 @@ public void end() {
         return label;
     }
 
+    /***
+     * Create a typing-label, except with a specified scriptName that corresponds to a fileName on the classpath,
+     * that will be loaded, and the typing label will be filled with it's contents.
+     *
+     * Intended to allow tests to be loaded from text files instead of hard coded.
+     *
+     * @return The typing label filled with the contents of the file at scriptName
+     */
+    public TypingLabel createTypingLabel(String scriptName) {
+
+        //Very strange replace needed, but it's needed. not sure if its my text file or a libgdx thing.
```

Comment:
It's your text file. You have literal `backslash, lower-case-n` pairs in your text file, which should be newlines instead. The line should probably be `String text = Gdx.files.internal(scriptName).readString();` and the text file fixed (just find any `\n` and replace it by hitting enter, if a regex replace isn't working). The text file may need to be saved with Unix line endings; I don't know if readString() handles `\r\n` as if it is `\n`.

Reply:
Cheers so much 😂, that was very silly. I did indeed open my text editor and understand what you meant.

I tested it and ensured that it reads correctly without the replace.